src/components/shared/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import { faFileAlt, faGlobeAmericas, faUsers, faTable, faChartBar } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useContext, useEffect, useState } from 'react'
+import { useMediaQuery } from 'react-responsive'
+import { Link, useHistory, useLocation } from 'react-router-dom'
+import MultiColorIcon from '../../assets/images/colored-icon.svg'
+import { mobileDeviceOrTabletWidth } from '../../constants'
+import Translate from '../../utils/translate/translateService'
+import { AppContext } from "../../context"
+import { withLocaleUrl } from "../../utils/utils";
+
+export const Footer = () => {
+  const [{updatedAt}] = useContext(AppContext);
+
+  const getTabClass = (tabName: string) => {
```

Comment:
Doesn't look like this func is doing anything, could we remove it?

Reply:
You right, was an artifact left behind from something i copied in. will remove